src/test/datascience/notebook/helper.ts

```diff
@@ -846,10 +869,17 @@ export async function hijackPrompt(
             ('contains' in message && msg.trim().includes(message.contains.trim())) ||
             ('endsWith' in message && msg.endsWith(message.endsWith))
         ) {
+            messageDisplayed.push(msg);
             traceInfo(`Exact Message found '${msg}'`);
             displayCount += 1;
             displayed.resolve(true);
             if (buttonToClick) {
+                if (!buttonToClick.dismissPrompt && buttonToClick?.clickImmediately === true && buttonToClick.text) {
```

Comment:
Tweeked this function so that we can use the same stub, but change what buttons are clicked.